packages/desktop-client/src/components/GlobalKeys.js

```diff
@@ -12,17 +12,17 @@ class GlobalKeys extends React.Component {
 
       if (e.metaKey) {
         const { history } = this.props;
-        switch (e.keyCode) {
-          case 49:
+        switch (e.code) {
+          case 'Digit1':
```

Comment:
Very excited about this change

Reply:
I'm quite sad I could not extract it outside of the React upgrade PR.. but there's lots of inter-connected dependencies that need to be put in place to finally move away from deprecated `keyCode`.